packages/browser-wallet-api-helpers/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 -   Expose a JSON-RPC client, using the wallet's current JSON-RPC server.
+-   `getMostRecentlySelectedAccount` method. This method allows dApps to get the most prioritized account without using `connect`.
```

Comment:
```suggestion
-   `getMostRecentlySelectedAccount` method. This method allows dApps to get the most prioritized account without using `connect`. In a future release it will be updated to actually return the most recently selected account.
```